index.html

```diff
@@ -66,8 +72,77 @@ <h3>Project 3</h3>
             </div>
         </section>
 
+        <!-- Skills section -->
+        <section class="skills">
```

Comment:
For the navbar link to work, we need to add `id="skills"` within this tag. Could you please incorporate that?